Batteries/Data/Thunk.lean

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
needs copyright header

Reply:
I asked @eric-wieser what I should put there. There seems to be no claim to this. I'll put my name for now and we can fix it later.